### PR TITLE
Issue-376

### PR DESCRIPTION
**Pull request by rgulrajani**
Updated UniversalAdIds for multiple UniversalAdId nodes

### Description

[VAST-Client] Allow parsing of multiple UniversalAdID

### Issue

https://github.com/dailymotion/vast-client-js/issues/376

### Type
- [x] Breaking change
- [x] Enhancement
- [ ] Fix
- [x] Documentation
- [ ] Tooling

### CODE REVIEW

File: docs/api/class-reference.md [hunk 1]
```diff
@@ -181,6 +181,11 @@ This object represents a generic Creative. It's used as a parent object for more
 - `attributes: Object`
 - `children: Array<Extension>`
 
+## UniversalAdId<a name="universaladid"></a>
+
+- `idRegistry: String|unknown`
+- `value: any`
```
Comment (ZacharieTFR) [inline docs/api/class-reference.md:187]: For `idRegistry` as the default value is a string, I think that we should put `String|"unknown"`  or add the description from IAB specs : `A string used to identify the URL for the registry website where the unique
creative ID is cataloged. Default value is “unknown”.` .
Also `value` is a String

File: src/vast_tracker.js [hunk 2]
```diff
@@ -612,13 +612,12 @@ export class VASTTracker extends EventEmitter {
     }
     if (
       this.creative &&
-      this.creative.universalAdId &&
-      this.creative.universalAdId.idRegistry &&
-      this.creative.universalAdId.value
+      this.creative.universalAdIds &&
+      this.creative.universalAdIds.length
```
Comment (ZacharieTFR) [inline src/vast_tracker.js:616]: What do you think of using the optional chaining operator for l.617 to 616 ? 
```suggestion
      this.creative?.universalAdIds?.length
```

**Review by ZacharieTFR — CHANGES_REQUESTED** [review]
Thank you for your contribution! 👍 I left some comments.

File: src/vast_tracker.js (hunk 2 above)
Comment (rgulrajani) [reply src/vast_tracker.js]: Updated

**Review by clementFrancon — APPROVED** [review]
LGTM ! Great job and thanks for contributing !

**Review by ZacharieTFR — APPROVED** [review]

**Review by rumesh — APPROVED** [review]
👍🏽

**Pull request merged**